### PR TITLE
updates to test trading

### PR DESCRIPTION
**Pull request by andrewdamelio**
more #3998 items
- made market categorical 
- hide Help Center link on tablet and below
- don't close Tutorial modal when syncing user data

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🥳

 This link will always point to this PR's latest changes: https://peek.run/zr5hep9

File: packages/augur-ui/src/modules/market/containers/market-view.ts [hunk 1]
```diff
@@ -34,27 +36,30 @@ const mapStateToProps = (state, ownProps) => {
...
           {
             avgPrice: formatDai(TUTORIAL_PRICE),
             cumulativeShares: TUTORIAL_QUANTITY.toString(),
             id: 1,
-            mySize: "0",
+            mySize: '0',
             orderEstimate: createBigNumber(.4),
             outcomeId: 2,
-            outcomeName: "Yes",
+            outcomeName: 'Los Angeles Rams',
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/market/containers/market-view.ts]: can these be constants?

File: packages/augur-ui/src/modules/market/containers/market-view.ts [hunk 2]
```diff
@@ -34,27 +36,30 @@ const mapStateToProps = (state, ownProps) => {
...
             price: TUTORIAL_PRICE.toString(),
             quantity: TUTORIAL_QUANTITY.toString(),
             shares: TUTORIAL_QUANTITY.toString(),
             sharesEscrowed: formatShares(TUTORIAL_QUANTITY),
             tokensEscrowed: formatDai(TUTORIAL_PRICE),
-            type: "buy",
+            type: 'buy',
```
Comment (phoebemirman) [inline packages/augur-ui/src/modules/market/containers/market-view.ts]: constant please

**Review by phoebemirman — CHANGES_REQUESTED** [review]
just a small change to use constants

**Review by phoebemirman — APPROVED** [review]

**Pull request merged**